presto-server-rpm/src/main/resources/dist/etc/init.d/presto

```diff
@@ -38,7 +38,7 @@ start () {
         echo "Warning: No value found for $PRESTO_JAVA_HOME. Default Java will be used."
         sudo -u $SERVICE_USER /usr/lib/presto/bin/launcher start "${CONFIGURATION[@]}"
     else
-        sudo -u $SERVICE_USER PATH=${$PRESTO_JAVA_HOME}/bin:$PATH /usr/lib/presto/bin/launcher start "${CONFIGURATION[@]}"
+        sudo -u $SERVICE_USER PATH=${PRESTO_JAVA_HOME}/bin:$PATH /usr/lib/presto/bin/launcher start "${CONFIGURATION[@]}"
```

Comment:
why RPM tests did not fail?

Reply:
Because it used default java, which was 11.